NEWS.md

```diff
@@ -0,0 +1,26 @@
+# data.validator 0.2.0
+
+-   `validate_cols()` will use all columns in dataframe if no column is passed
+-   Added a defensive wrapper (`data.validator:::check_assertr_expression()`) to add evaluation error to regular validation errors
+-   New examples added:
+    -   Shiny App example demo example added
+    -   Usage examples added in README
+    -   Connect workflow examples added
+-   pkgdown documentation updated
+
+# data.validator 0.1.6
+
+-   README updated
+-   R version updated
+    -   windows: `3.6` to `4.1.2`
+    -   macOS: `4.0.2` to `4.1.2`
+    -   ubuntu 18.04: `3.6` to `4.1.2`
```

Comment:
This is not a change that affects the code. Version 0.1.6 was pushed to CRAN only to restore `data.validator` (after one of its dependencies was removed) and there were no actual changes implemented. The changelog should reflect that.